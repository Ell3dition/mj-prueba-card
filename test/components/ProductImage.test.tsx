import React from "react"

import renderer from "react-test-renderer"
import ProductCard, { ProductImage } from "../../src/components"
import { productDos } from "../data/product"

describe('Product Image', () => {



    test('debe mostrar componente renderizado ', () => {

        const wrapper = renderer.create(
            <ProductImage img="./coffe-mug.png" className="custom-class" />
        )
        
        expect(wrapper.toJSON()).toMatchSnapshot()
      
    })
    
    test('debe de mostrar la imagen del producto', () => {
        const wrapper = renderer.create(
            <ProductCard product={productDos} >
                {
                    ()=>(
                        <ProductImage />

                    )
                }
            </ProductCard>
        )
        
        expect(wrapper.toJSON()).toMatchSnapshot()
    })
  
})
