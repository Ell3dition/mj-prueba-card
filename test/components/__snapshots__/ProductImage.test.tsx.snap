// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Image debe de mostrar la imagen del producto 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`Product Image debe mostrar componente renderizado  1`] = `
<img
  alt="product"
  className="productImg custom-class"
  src="./coffe-mug.png"
/>
`;
